frontend/src/scenes/events/eventsTableLogic.test.ts

```diff
@@ -340,13 +412,21 @@ describe('eventsTableLogic', () => {
                         })
                         logic.actions.fetchNextEvents()
                     }).toDispatchActions([
-                        logic.actionCreators.fetchEvents({ before: secondEvent.timestamp }),
+                        logic.actionCreators.fetchEvents({
+                            before: secondEvent.timestamp,
+                            after: afterLastEventsTimestamp,
```

Comment:
need one more test here to confirm before isn't changed, after becomes aYearAgo, when "load more" is clicked, but no events in past 5 days.